doc/reference/std/html.md

```diff
@@ -0,0 +1,188 @@
+# HTML Parser and Printer
+
+HTML is a widely used Markup Language that, while very similar to [XML](./xml.md), differs enough to have its own specific libraries.
+
+*Usage*
+
+```scheme
+(import :std/html)
+```
+
+This is for parsing and printing HTML. For HTML usage and templates for web development have a look at [The Template Attribute Language (TAL)](tal/README.md).
+
+```scheme
```

Comment:
Who are these lines for? That's confusing.

The context suggests it might be something for the users, but the content suggests it's only for implementers.

Reply:
None of my documentation is meant for implementers of what I'm documenting. Can tell me where the content points towards implementation over usage? Will certainly change. 

The lines are meant to point to other documentation that's related to what is being discussed. I can clear up the confusion with some pointers.